src/evo/mnhftx.h

```diff
@@ -19,20 +19,17 @@ extern RecursiveMutex cs_main;
 class MNHFTx
 {
 public:
-    static constexpr uint16_t LEGACY_BLS_VERSION = 1;
-    static constexpr uint16_t BASIC_BLS_VERSION = 2;
-
-    uint16_t nVersion{LEGACY_BLS_VERSION};
+    uint8_t versionBit{0};
```

Comment:
I like that, should help avoiding confusion. Should also update DIP accordingly.

Reply:
https://github.com/dashpay/dips/pull/138